test-launcher.py

```diff
@@ -34,9 +38,10 @@ def header_text(t):
 repo = sys.argv[1]
 
 def container_run(repo):
-  r = os.path.join(os.getcwd(), 'repo') + ':/build:Z'
-  f = os.path.join(os.getcwd(), 'features') + ':/behave:Z'
-  DOCKER_RUN = ['docker', 'run', '-i', '-v', r, '-v', f, DOCKER_IMAGE, repo]
+  r = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'repo') + ':/build:Z'
```

Comment:
`work_dir = os.path.dirname(os.path.realpath(__file__)` and then reuse the variable?